Packs/EDL/Integrations/EDL/README.md

```diff
@@ -72,7 +72,16 @@ These parameters prevent the integration from incorrectly inserting unwanted TLD
 The default value for `Maximum CIDR network prefix bits size` is 8, which means that CIDRs with a lower network prefix bits number are not included (such as 0.0.0.0/2).
 
 
-The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as \*.com, \*.co.uk, \*.org and other top level domains.
+The default value for `Exclude top level domainGlobs` is off. If enabled, the exported list does not hold indicators such as `*.com`, `*.co.uk`, `*.org` and other top level domains.
+
+### DomainGlobs
+
+When parsing DomainGlobs the parser will create two different inputs as that is usually how DNS Firewalls work. For example if the DomainGlob `*.bad.com` will be parsed it will output two lines to the list:
```

Comment:
looks like the whole document refers to this as domainGlobs (small d) so i think it's best to keep that aligned here as well

Reply:
also - i might say 'when parsing domainGlob indicator types' - to make it explicit that it's a type-related behavior.

Reply:
capital D is my bad there.